extensions/amp-social-share/1.0/amp-social-share.css

```diff
@@ -20,6 +20,16 @@
   * See https://github.com/ampproject/amphtml/issues/4277 for more details.
   */
 
+amp-social-share:focus-within {
+  outline: #0389ff solid 2px;
+  outline-offset: 2px;
+}
+
+amp-social-share::part(button):focus {
```

Comment:
> Not sure how would we overwrite in preact mode? Inline style does not seem to be able to control :focus pseudo selector

It looks like this section is doing the overwriting portion. Is this open question still applicable?

Reply:
Right, this is definitely overwriting in amp mode.  I was just thinking in Preact mode, seems like we have been passing in a style object (as a prop) to customize things in most components (or in the case of social-share, we use the `color` and `background` props since there isn't much else to customize).

But I was unsure how we might do this for focusable styles, since I don't think you can have a style only refer to the `focus` psuedo selector when passing it inline.

Maybe there's an easy way that I am not aware of though?